### PR TITLE
VP-1379: Switch to typescript

### PR DESCRIPTION
**Pull request by asvishnyakov**

### CODE REVIEW

File: client-app/src/pages/account/views/Home.vue [hunk 1]
```diff
@@ -1,88 +1,67 @@
...
-    }
-};
+@Component({
+  name: "ProfileHome",
+  components: {
+    Loading
+  }
+})
+export default class Home extends Vue {
+  get profile() {
```
Comment (tatarincev) [inline client-app/src/pages/account/views/Home.vue]: Is this syntax sugar won't work here? ` ...mapGetters(["profile", "isLoading"])`

File: client-app/src/i18n.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Vue.use(VueI18n);
+
+const i18n = new VueI18n({
+  locale: process.env.VUE_APP_I18N_LOCALE || "en",
+  fallbackLocale: process.env.VUE_APP_I18N_FALLBACK_LOCALE || "en",
+  messages: {}
+});
+
+LocalizationService.get().then(({ data }) => {
```
Comment (tatarincev) [inline client-app/src/i18n.ts]: I don't remember exactly why but  `i18n.setLocaleMessage("en", data);` didn't work before in this place. All your resources were untranslated.

File: client-app/src/pages/account/views/Home.vue (hunk 1 above)
Comment (asvishnyakov) [reply client-app/src/pages/account/views/Home.vue]: No and it shouldn't. You loose type checking when you use strings

File: client-app/src/i18n.ts (hunk 2 above)
Comment (asvishnyakov) [reply client-app/src/i18n.ts]: Will check

Comment (asvishnyakov) [reply client-app/src/i18n.ts]: @tatarincev Working perfectly:
image
image

**Pull request merged**